### PR TITLE
simplify email regex

### LINKED ISSUE #13: support long TLDs in emails
Current email validator only supports 2-3 character long TLDs.

### PR DESCRIPTION
**Pull request by tanner0101**
fixes #13

### CODE REVIEW

File: Sources/Validation/Convenience/Email.swift [hunk 1]
```diff
@@ -3,15 +3,12 @@ import Foundation
 public struct EmailValidator: Validator {
 
     public init() {}
-	
-	private static let name = "[a-z0-9!#$%&'*+/=?^_`{|}~]"
```
Comment (Joannis) [inline Sources/Validation/Convenience/Email.swift:7]: Were these special characters not technically correct? I've seen people with special characters in their email such as: "ç", "å", "µ", "∆". think of Greece, Sweden, Spain, Russia and some of the arabic speaking countries

Comment (tanner0101) [reply Sources/Validation/Convenience/Email.swift:7]: even as it was previously the characters you put would be marked incorrect. maybe we should look at what other frameworks do? the only issue seems to be the length of the TLD.

Comment (Joannis) [reply Sources/Validation/Convenience/Email.swift:7]: We could invert this. Accept all characters _except_ `@`, `.`, ...

**Review by vzsg — APPROVED** [review]

**Review by 0xTim — APPROVED** [review]
Looks good to allow longer TLDs

**Pull request merged**